desktop/apps/categories2/components/AppContainer.js

```diff
@@ -3,36 +3,82 @@ const data = require('../data/data.json')
 
 import React from 'react'
 import styled from 'styled-components'
-import { media } from '../styles/style-utils';
+import { media } from '../styles/style-utils'
 
 import Intro from './Intro'
-import Sidebar from './Sidebar'
-import Content from './Content'
-import GeneFamilies from './GeneFamilies'
+import FamilyList from './FamilyList'
+import FamilyAndGeneList from './FamilyAndGeneList'
+
+const StyledIntro = styled(Intro)`
+  font-size: 20px;
+  line-height: 1.33em;
+  p {
+    margin: 1em 0;
+  }
+  h2 {
+    font-size: 50px;
+    line-height: 100%;
+  }
+`
+
+const StyledFamilyList = styled(FamilyList)`
+  font-size: 14px;
+  color: #333;
+  text-transform: uppercase;
+`
+
+const StyledFamilyAndGeneList = styled(FamilyAndGeneList)`
+  .gene-family {
+    border-top: 1px solid #ccc;
+    margin: 1em 0;
+    padding: 1em 0;
+  }
+  h3 {
+    font-size: 25px;
+    line-height: 1.4em;
+  }
+  ul {
+    margin: 1em 0;
+  }
+  ${media.mobile`
+    ul {
+      column-count: 3;
+      column-gap: 1em;
+    }
+  `}
+`
+
+const Grid = styled.div`
+  padding: 40px 0;
+  display: flex;
+  flex-direction: column;
+  ${media.mobile`
+    flex-direction: row;
+  `}
+`
 
 const Column = styled.div`
-  ${ media.mobile`
-    display: flex
-    flex: 0 1 auto
-    flex-flow: row wrap
-  ` }
+  flex: 0 0 ${props => props.width * (100.0 / 12)}%;
```

Comment:
Theoretically we could make the number of columns in layout to be flexible as well, and pass it as a parameter, right?